moveit_core/robot_model/src/joint_model_group.cpp

```diff
@@ -216,7 +208,7 @@ JointModelGroup::JointModelGroup(const std::string& group_name, const srdf::Mode
     | ranges::to<std::vector>();
   active_joint_model_start_index_ =
     views::zip(start_index_view, not_fixed_view)
-    | views::remove_if([](const auto& e) { return is_mimic_joint(std::get<1>(e)); })
+    | views::remove_if([=](const auto& e) { return is_mimic_joint(std::get<1>(e)); })
```

Comment:
What is getting copied here?

Reply:
The is_mimic_joint lambda.  For some reason clang won't capture it by default where gcc will.